### PR TITLE
TimezoneTimeInfo 

### PR DESCRIPTION
**Pull request by tn3rb**
while troubleshooting some timezone offset issues I wanted a way to see dates in the following timezones:

- the users local timezone
- the timezone set for the site
- UTC

added a component for displaying this info in a non-obtrusive way to the biggie and medium calendar date components
also converted those to BEM

here's a screenie:

Timezone TimeInfo

### CODE REVIEW

File: assets/src/application/services/hooks/useTimeZoneTime.ts [hunk 1]
```diff
@@ -1,29 +1,103 @@
...
+	 * returns a date string formatted using the user's locale but UTC timezone
+	 */
+	const formatUtcDateForUser: FormatDateFn = (date, options) => {
+		const dateObject = date instanceof Date ? date : parseISO(date);
+		if (!isValid(dateObject)) {
+			return null;
+		}
+		const formatOptions = { ...deafultOptions, ...options };
+		return dateObject.toLocaleString(userLC, { ...formatOptions, timeZone: 'UTC' });
+	};
```
Comment (manzoorwanijk) [inline assets/src/application/services/hooks/useTimeZoneTime.ts]: Nice 👍

File: assets/src/application/valueObjects/config/Locale.ts [hunk 2]
```diff
@@ -1,8 +1,10 @@
...
 export const Locale = ({ user, site }: LocaleProps): LocaleProps => {
+	const userLocale = user || 'en-US';
+	const siteLocale = site || 'en-US';
 	return {
-		user: user || 'en_US',
-		site: site || 'en_US',
+		user: userLocale.replace('_', '-'),
+		site: siteLocale.replace('_', '-'),
 	};
 };
```
Comment (manzoorwanijk) [inline assets/src/application/valueObjects/config/Locale.ts]: What about this?
```ts
	const userLocale = user?.replace('_', '-') || 'en-US';
	const siteLocale = site?.replace('_', '-') || 'en-US';
	return {
		user: userLocale,
		site: siteLocale,
	};
```

Comment (tn3rb) [reply assets/src/application/valueObjects/config/Locale.ts]: coll ... will do

File: assets/src/application/ui/calendars/dateDisplay/BiggieCalendarDate/style.scss [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+		width: 100%;
+		vertical-align: bottom;
+	}
+
+	&__header {
+		font-size: var(--ee-font-size-micro);
+		letter-spacing: var(--ee-letter-spacing-font-size-micro);
+		line-height: var(--ee-font-size-bigger);
+		margin: 0 0 var(--ee-margin-micro);
+	}
```
Comment (nicolad) [inline assets/src/application/ui/calendars/dateDisplay/BiggieCalendarDate/style.scss]: neeeeat :+1: :taco:

File: assets/src/application/ui/calendars/dateDisplay/BiggieCalendarDate/style.scss [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+
+	.ee-bcd {
```
Comment (nicolad) [inline assets/src/application/ui/calendars/dateDisplay/BiggieCalendarDate/style.scss]: what is `bcd`?

**Pull request merged**